org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.full/org.xtext.example.full.parent/pom.xml

```diff
@@ -12,7 +12,7 @@
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<!-- Tycho settings -->
-		<tycho-version>1.2.0</tycho-version>
+		<tycho-version>1.3.0-SNAPSHOT</tycho-version>
```

Comment:
Can we encapsulate all these changes in a single pom / bom?

Reply:
if you spot correct: these a n different testcases with 1 change each